info.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "connectorHealthMonitoring",
+    "version": "1.0.0",
+    "type": "solutionpack",
+    "local": true,
+    "label": "Connector Health Monitoring",
+    "dependencies": [],
+    "prerequisite": null,
+    "publisher": "Fortinet",
+    "certified": "true",
+    "description": "Solution Pack send the connector health status notification via email based on provided connector list.",
+    "help": null,
+    "category": [],
```

Comment:
Category should be 'health monitoring' or so.. @cs-rohit-khune please provide your guidance here

Reply:
@cs-suyog-jinturkar  Discussed with Rohit, we have two category as "Monitoring" and "System Monitoring". So which one we should pick for this SP?

Reply:
Monitoring as this is not related to sysmon